Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/CalculateMonteCarloAbsorptionTest.py

```diff
@@ -0,0 +1,144 @@
+from mantid.kernel import *
+from mantid.api import *
+from mantid.simpleapi import (Load, DeleteWorkspace, CalculateMonteCarloAbsorption)
+
+import unittest
+
+class CalculateMonteCarloAbsorptionTest(unittest.TestCase):
+
+    _red_ws = None
+    _container_ws = None
+    _indirect_elastic_ws = None
+
+    def setUp(self):
+        red_ws = Load('irs26176_graphite002_red.nxs')
+        self._red_ws = red_ws
+        self._expected_unit = self._red_ws.getAxis(0).getUnit().unitID()
+        self._expected_hist = 10
+        self._expected_blocksize = 1905
+
+        self._arguments = {'SampleChemicalFormula': 'H2-O',
+                           'SampleDensityType': 'Mass Density',
+                           'SampleDensity': 1.0,
+                           'EventsPerPoint': 200,
+                           'BeamHeight': 3.5,
+                           'BeamWidth': 4.0,
+                           'Height': 2.0 }
+        self._test_arguments = dict()
+
+    def tearDown(self):
+        DeleteWorkspace(self._red_ws)
+
+        if self._container_ws is not None:
+            DeleteWorkspace(self._container_ws)
+        if self._indirect_elastic_ws is not None:
+            DeleteWorkspace(self._indirect_elastic_ws)
+
+    def _setup_container(self):
+        container_ws = Load('irs26173_graphite002_red.nxs')
```

Comment:
You call this twice, once for each shape, but you should load only once. I would move these to `setUp` just like yo do with the `red_ws`.